classes/test/StrictMathTest.java

```diff
@@ -261,15 +268,6 @@ public static void main(String[] args) {
     }
     results("double pow(double a, double b)", d_results_2d);
 
-    // static double hypot(double a, double b)
-    // Returns the value of the first argument raised to the power of 2 + second argument raised to the power of .
-    for (i=0; i < d_vals.length; i++) {
```

Comment:
Actually I have no idea how this happened ! sorry ! is the rest good ?

On Tue, Mar 10, 2015 at 12:30 AM, CJ Carey <notifications@github.com> wrote:

> In classes/test/StrictMathTest.java
> <https://github.com/plasma-umass/doppio/pull/365#discussion_r26064496>:
>
> > @@ -261,15 +268,6 @@ public static void main(String[] args) {
> >      }
> >      results("double pow(double a, double b)", d_results_2d);
> >
> > -    // static double hypot(double a, double b)
> > -    // Returns the value of the first argument raised to the power of 2 + second argument raised to the power of .
> > -    for (i=0; i < d_vals.length; i++) {
>
> Why is this getting deleted?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plasma-umass/doppio/pull/365/files#r26064496>.
>